app/public/js/localSprites.js

```diff
@@ -112,18 +115,18 @@ function updateLocalSprite(localSprite) {
         }
     }
     //contrail generation
-    //TOO SPECIFIC TO FIREBALL RIGHT NOW. It seems like you could give addSprite functions a bunch of new parameters to fix this, but that seems messy, as a lot of sprites won't have contrails.
+    //WRONG: TOO SPECIFIC TO FIREBALL RIGHT NOW. It seems like you could give addSprite functions a bunch of new parameters to fix this, but that seems messy, as a lot of sprites won't have contrails.
     if (localSprite.hasContrail === true) {
         if (localSprite.contrailTimer >= localSprite.framesBetweenContrailParticles) {
-            addFireballContrailParticle(localSprite, 30, 32, 32);
+            addFireballParticle(localSprite, 30, 32, 32, 0); //for last argument, 0 = contrail, 1 = detonation
             localSprite.contrailTimer = 0;
         } else {
             localSprite.contrailTimer++;
         }
     }
     localSprite.x += localSprite.vx;
     localSprite.y += localSprite.vy;
-    //max speed limit:
+    //max speed limit: Chris is replacing max speed with just friction vs. acceleration in some places, but here accel affects maneuverability and speed needs to be carefully controlled, so maybe use max speed in this case? Or maybe it would work out otherwise.
```

Comment:
Since I just removed it from `updateActor` and fireballs use a different update function, you are free to keep using max speed here.